integrations-and-supported-tools/detectron2/notebooks/Neptune_detectron2.ipynb

```diff
@@ -413,7 +424,7 @@
     "    out = v.draw_instance_predictions(outputs[\"instances\"].to(\"cpu\"))\n",
     "    image = out.get_image()[:, :, ::-1]\n",
     "    img_rgb = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)\n",
-    "    run[\"training/prediction_visualization\"][idx].upload(\n",
+    "    run[\"training/prediction_visualization\"][f\"{idx}\"].upload(\n",
```

Comment:
Why is this change needed?

Reply:
It leads to error
```
Traceback (most recent call last):
  File "Neptune_detectron2.py", line 141, in <module>
    run["training/prediction_visualization"][idx].upload(File.as_image(img_rgb / 255.0))
  File "/opt/conda/lib/python3.8/site-packages/neptune/handler.py", line 114, in __getitem__
    return Handler(self._container, join_paths(self._path, path))
  File "/opt/conda/lib/python3.8/site-packages/neptune/internal/utils/paths.py", line 34, in join_paths
    return "/".join(_remove_empty_paths(list(paths)))
TypeError: sequence item 1: expected str instance, int found
```

Reply:
I was under the impression that neptune would take care of converting it to a string.
In this case, lets stick to `run[f"training/prediction_visualization/{idx}"]`